bedrock/base/views.py

```diff
@@ -3,12 +3,30 @@
 
 from django.conf import settings
 from django.http import HttpResponse, HttpResponseBadRequest
+from django.views.decorators.cache import never_cache
 from django.views.decorators.csrf import csrf_exempt
-from django.views.decorators.http import require_POST
+from django.views.decorators.http import require_POST, require_safe
+
+from bedrock.mozorg.util import HttpResponseJSON
+from lib import l10n_utils
 
 from raven.contrib.django.models import client
 
-from lib import l10n_utils
+
+@require_safe
+@never_cache
+def geolocate(request):
+    """Return the country code provided by our CDN
+
+    https://support.cloudflare.com/hc/en-us/articles/200168236-What-does-CloudFlare-IP-Geolocation-do-
+    """
+    country_code = request.META.get('HTTP_CF_IPCOUNTRY', 'XX')
+    if country_code == 'XX' or len(country_code) != 2:
+        return HttpResponseJSON({'country_code': None}, status=404)
+
+    return HttpResponseJSON({
+        'country_code': country_code,
+    })
```

Comment:
Wondering if we could put an easy short-circuit here for local dev. Maybe have it look for a config setting? Or if `DEBUG=True` just return `us`?